yaml/wp_base.yaml

```diff
@@ -270,6 +276,7 @@ packages:
   SPEICHERISTTEMP:                !include { file: wp_temperature.yaml, vars: { property: "SPEICHERISTTEMP"          , interval: $interval_medium}}
   AUSSENTEMP:                     !include { file: wp_temperature.yaml, vars: { property: "AUSSENTEMP"       }}
   ABLUFTTEMP:                     !include { file: wp_temperature.yaml, vars: { property: "ABLUFTTEMP"       }}
+  ABLUFTTAUPUNKT:                 !include { file: wp_temperature.yaml, vars: { property: "ABLUFTTAUPUNKT"   }}
```

Comment:
Can you please add this in a separate commit together with the DIFFERENZDRUCK and only for the thz504.yaml until it is confirmed by all variants that it works?